File: spring-ai-a2a-examples/java/v2/order-agent/src/main/java/io/github/timo/a2a/spring/ai/agent/example/OrderAgentApplication.java
package io.github.timo.a2a.spring.ai.agent.example;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class OrderAgentApplication {

    public static void main(String[] args) {
        SpringApplication.run(OrderAgentApplication.class, args);
    }
}

File: spring-ai-a2a-examples/kotlin/v2/order-agent/src/main/kotlin/io/github/timo/a2a/spring/ai/agent/example/OrderAgentApplication.kt
package io.github.timo.a2a.spring.ai.agent.example

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class OrderAgentApplication

fun main(args: Array<String>) {
    runApplication<OrderAgentApplication>(*args)
}
